src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -4,7 +4,8 @@ import {FlatList as NativeFlatlist, View} from 'react-native';
 import _ from 'underscore';
 import FlatList from '@components/FlatList';
 import * as CollectionUtils from '@libs/CollectionUtils';
-import variables from '@styles/variables';
+
+const AUTOSCROLL_TO_TOP_THRESHOLD = 128;
```

Comment:
This PR forgot to cover an extra case. More details here https://github.com/Expensify/App/issues/30726